api/server/sdk/objectstore.go

```diff
@@ -68,6 +68,14 @@ func (s *ClusterServer) UpdateObjectstore(
 	req *api.SdkObjectstoreUpdateRequest,
 ) (*api.SdkObjectstoreUpdateResponse, error) {
 
+	err := s.cluster.ObjectStoreUpdate(req.GetObjectstoreId(), req.GetEnable())
```

Comment:
Since ObjectstoreId is optional here avoiding check for it. 
Also relying on default value provided by protobuf
- for string it's  empty string 
- for bool it's false by default
@lpabon Let me know is it ok to rely on default value generated by api.proto keeping in mind of future sdk release